src/views/AgmLocationChg.vue

```diff
@@ -642,10 +638,12 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     }
   }
 
-  @Watch('agmYearValid')
-  @Watch('agmLocationValid')
-  @Watch('certifyFormValid')
-  @Watch('resonValid')
+  /** Watches all data properties to keep track of changes. */
+  @Watch('agmYear')
+  @Watch('agmLocation')
+  @Watch('certifiedBy')
+  @Watch('isCertified')
+  @Watch('reason')
```

Comment:
Why did you change these Sev?

Reply:
To directly check if the user has changed any input fields.

Reply:
For example, if the user entered certifiedBy but didn't check isCertified, the certify form would be invalid, but they have in fact entered some data.

Reply:
Got it, thanks! Resolving.